source/process/documentation-UItext-guidelines.rst

```diff
@@ -0,0 +1,183 @@
+
+....
+
+.. contents:: 
+   :depth: 2
+
+....
+
+UI elements
+------------
+Use this table when writing the text for UI elements such as windows and dialog boxes.
+
+.. list-table::
+   :widths: 10 30 10 15 35
+   :header-rows: 1
+
+   * - Element
+     - Image 
+     - Capitalization
+     - Phrasing
+     - Examples     
+   * - Menu
+     - |menu.png|
+     - Headline
+     - - Noun, noun phrase, or verb
+       - No punctuation
+       - Not more than three words
+     - - *Members*
+       - *Account Preferences*
+       - *Log Out*
+   * - Tooltip
+     - |tooltip.png|
+     - Sentence
+     - - Sentence fragment or sentence
+       - No punctuation
+       - Include articles (*a*, *an*, *the*)
+     - - *Start a Zoom meeting*
+       - *Flag for following up*
+       - *Remove from this list*
+   * - In-field text
+     - |field.png|  
+     - Sentence
+     - - Sentence fragment, sentence, or word
+       - No punctuation
+       - Include articles (*a*, *an*, *the*)
+     - - *Add a comment*
+       - *Search*
+   * - Action button
+     - |action.png|
+     - Headline
+     - - Verb or verb phrase
+       - No articles (*a*, *an*, *the*)
+       - Exceptions: *OK*, *Yes*, *No*
+     - - *Add Comment*
+       - *Edit*
+   * - Label before a UI element
+     - |label_before.png|
+     - Sentence
+     - - Noun, verb, or sentence fragment
+       - End with a colon
+       - Include articles (*a*, *an*, *the*)
+     - - *Sign in with:*
+       - *Other words, separated by commas:*
+   * - Label after a UI element
+     - |label_after.png|
+     - Sentence
+     - - Noun, verb, or sentence fragment
+       - No punctuation
+     - - *Channels grouped by type*
+       - *Alphabetically*
+   * - Help text
+     - |help.png|
+     - Sentence
+     - Complete sentences, with punctuation
+     - - *You can add 20 more people.*
+       - *People are invited automatically to join the channel.*
+   * - Title
+     - |title.png|
+     - Headline
+     - - Sentence fragment or sentence
+       - No punctuation
+     - - *Notification Preferences for Channel*
+       - *Contributors*
+
+Messages
+--------
+
+In general, do not blame the user. Inform, explain, and suggest. 
+
+Try to use the active voice, except in these cases:
+
+- You'll end up blaming the user. For example, don't say *You entered an incorrect password*. Instead, say *The password is incorrect*.
+- You're describing what just happened. For example, it's okay to say *Your incoming webhook has been set up*.
+- The subject (the doer of an action) is the Mattermost application itself. For example, it's okay to say *The image was deleted* instead of *The server deleted the image*.
+
+Use complete sentences. If you're asking the user to do something, use imperatives. Use the past tense to inform what's already happened, the present tense to describe a current state or condition, and the future tense to state something that will definitely happen. 
```

Comment:
Can we define what imperatives are to make this more clear to the reader?

Reply:
I added an example and a tiny parenthetical infobyte for imperatives. Works?